bulletin_board/server/package.json

```diff
@@ -37,7 +37,7 @@
     "eslint-plugin-jasmine": "^4.1.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
-    "prettier": "2.1.2",
+    "prettier": "3.0.3",
```

Comment:
Are you changing this because there was some error in the pipeline related to the old version of prettier? I say because I prefer to separate the linter changes from the other changes, as they bring too much noise to the reviews.

Reply:
Okey makey!